### ISSUE TITLE
Unable to specify manageTrailingCommas option for Ktfmt

### ISSUE THREAD

**Issue by AlexandrosAlexiou** [diffplug/spotless#2348]
Hello!

I am using the maven plugin v.2.43.0 with ktfmt

When I specify `<manageTrailingCommas>true</manageTrailingCommas>` the `mvn spotless:apply` command fails with:

`Unable to parse configuration of mojo com.diffplug.spotless:spotless-maven-plugin:2.43.0:apply for parameter manageTrailingCommas: Cannot find 'manageTrailingCommas' in class com.diffplug.spotless.maven.kotlin.Ktfmt `

Here is the whole config:

```xml 
            <plugin>
                <groupId>com.diffplug.spotless</groupId>
                <artifactId>spotless-maven-plugin</artifactId>
                <version>2.43.0</version>
                <configuration>
                    <kotlin>
                        <ktfmt>
                            <style>KOTLINLANG</style>
                            <manageTrailingCommas>true</manageTrailingCommas>
                        </ktfmt>
                    </kotlin>
                </configuration>
            </plugin>
```

**Comment by EniXoo** [comment]
Hey ! I faced the same issue 👍 

I've decided to check the README of the [v2.43.0 ](https://github.com/diffplug/spotless/tree/maven/2.43.0/plugin-maven#ktfmt) and it seems that it is not implemented on this version 😃 It seems that you'll need at least [v2.44.0BETA2](https://github.com/diffplug/spotless/tree/maven/2.44.0.BETA2/plugin-maven#ktfmt) 😉

Hoping it will help you 👌

**Comment by AlexandrosAlexiou** [comment]
Yeah, I already used [v2.44.0BETA2](https://github.com/diffplug/spotless/tree/maven/2.44.0.BETA2/plugin-maven#ktfmt) and it works for now, but thanks for the info! 👍
